cmd/export_contract_events.go

```diff
@@ -0,0 +1,67 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/stellar/stellar-etl/internal/input"
+	"github.com/stellar/stellar-etl/internal/transform"
+	"github.com/stellar/stellar-etl/internal/utils"
+)
+
+var contractEventsCmd = &cobra.Command{
+	Use:   "export_contract_events",
+	Short: "Exports the contract events over a specified range.",
+	Long:  `Exports the contract events over a specified range to an output file.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		cmdLogger.SetLevel(logrus.InfoLevel)
+		cmdArgs := utils.MustFlags(cmd.Flags(), cmdLogger)
+
+		// TODO: https://stellarorg.atlassian.net/browse/HUBBLE-386 GetEnvironmentDetails should be refactored
```

Comment:
I left a bunch of TODOs on purpose. It didn't seem right to refactor everything in this same PR